src/helm/proxy/clients/openai_client.py

```diff
@@ -53,13 +53,25 @@ def make_request(self, request: Request) -> RequestResult:
                 "engine": request.model_engine,
             }
         elif self._is_chat_model_engine(request.model_engine):
-            raw_request = {
-                "model": request.model_engine,
+            messages: Optional[List[Dict[str, str]]] = request.messages
+            if request.messages and len(request.messages) > 1:
```

Comment:
Could we do some validation that only one of `messages` and `prompt` is set i.e. the user doesn't set both at once?